WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -444,14 +458,14 @@ class PagesViewModel
         }
     }
 
-    fun onDeleteConfirmed(remoteId: Long) {
-        launch(defaultDispatcher) {
-            pageMap[remoteId]?.let { deletePage(it) }
-        }
-    }
-
     fun onItemTapped(pageItem: Page) {
-        _editPage.postValue(pageMap[pageItem.id])
+        val page = postStore.getPostByRemotePostId(pageItem.id, site)
```

Comment:
We might want to consider adding a incode TODO here and link https://github.com/wordpress-mobile/WordPress-Android/issues/11233. Wdyt?

Reply:
Resolved in 155b6dd